lib/AST/StructFieldRenamer.cpp

```diff
@@ -60,6 +60,9 @@ bool StructFieldRenamer::VisitRecordDecl(clang::RecordDecl *decl) {
     // FIXME(frabert): Is a clash between field names actually possible?
     // Can this mechanism actually be left out?
     auto name{di_field->getName().str()};
+    if (di_field->getTag() == llvm::dwarf::DW_TAG_inheritance) {
```

Comment:
Does this make an explicit field out of the base type in the case of inheritance? Can you add a comment here that shows what a simple c++ code would look like, and what we would generate as a result?

Reply:
Can you add some examples that use multiple inheritance and virtual inheritance?

```c++
struct Base1 {
  int foo;
};
struct Base2 {
  float bar;
}
struct Derived : Base1 , Base2 {
};
```

```c++
struct Base1 {
  int foo;
};
struct Base2 : Base1 {
  float bar;
};
struct Base3 : Base1 {
  float bar;
};
struct Derived : virtual Base2 , virtual Base3 {
};
```

Also, here is a particularly thorny example which shows when this method of embedding the base within the structure of the parent is going to break down:
C++: https://godbolt.org/z/bM4vrq6fW
C: https://godbolt.org/z/fYarYo5he

See this SO post for more detail: https://stackoverflow.com/questions/52818411/will-the-padding-of-base-class-be-copied-into-the-derived-class